src/mame/atari/wolfpack.h

```diff
@@ -97,7 +97,7 @@ class wolfpack_state : public driver_device
 	uint8_t m_ship_h = 0;
 	uint8_t m_torpedo_pic = 0;
 	uint8_t m_ship_size = 0;
-	uint8_t m_ship_h_precess = 0;
+	uint8_t m_ship_h_process = 0;
```

Comment:
Are you sure this isn’t supposed to be short for “precession”?  It’s a real word.